google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/internal/AbstractProtoStructReader.java

```diff
@@ -166,14 +168,26 @@ public boolean getBoolean(String columnName) {
   }
 
   @Override
-  public Instant getTimestamp(int columnIndex) {
+  @ObsoleteApi("Use getTimestampInstant(int) instead")
+  public org.threeten.bp.Instant getTimestamp(int columnIndex) {
```

Comment:
If possible, add javadocs: `This method is obsolete. Use {@link ... `

Reply:
Done